ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/components/ExceptionRequestModal/ExceptionScopeField.tsx

```diff
@@ -1,23 +1,35 @@
 import React from 'react';
 import { FormGroup, FormGroupProps, Radio } from '@patternfly/react-core';
 
-import { ScopeContext } from './utils';
+import { useFormik } from 'formik';
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+import { DeferralValues, ScopeContext } from './utils';
+
+const ALL = '.*';
```

Comment:
What are pro and con for `getDefaultValues` function in DeferralForm.tsx file to use `ALL` constant?

Reply:
No cons that I can think of, I'll update so this stays consistent.